core/lib/globals_shared.h

```diff
@@ -928,6 +928,7 @@ typedef enum {
  */
 enum {
     NUDGE_IS_INTERNAL = 0x01, /* nudge is internally generated */
+    NUDGE_IS_SUSPEND = 0x02,  /* This is an internal SUSPEND_SIGNAL. */
 #ifdef WINDOWS
     NUDGE_NUDGER_FREE_STACK = 0x02, /* nudger will free the nudge thread's stack so the
                                      * nudge thread itself shouldn't */
```

Comment:
This goes into nudge_arg_t.flags right? But that field is only 2 bits and here we have 0x04 which won't fit.

Reply:
The flags field is 32 bits on Windows.

Reply:
(There is already a reminder comment for the 2 bits just on Linux right above this enum)